x/axelarnet/client/cli/tx.go

```diff
@@ -172,13 +175,39 @@ func GetCmdRegisterAsset() *cobra.Command {
 		}
 		chain := args[0]
 		denom := args[1]
+		limit := utils.MaxUint
+		window := types.DefaultRateLimitWindow
 
 		isNativeAsset, err := cmd.Flags().GetBool(flagIsNativeAsset)
 		if err != nil {
 			return err
 		}
 
-		msg := types.NewRegisterAssetRequest(cliCtx.GetFromAddress(), chain, nexus.NewAsset(denom, isNativeAsset))
+		limitArg, err := cmd.Flags().GetString(flagLimit)
+		if err != nil {
+			return err
+		}
+		if limitArg != "" {
+			l, ok := sdk.NewIntFromString(limitArg)
```

Comment:
why not use `sdk.NewUintFromString` directly?

Reply:
it seemed like the method didn't exist when i tried it, but looks like i missed something